assets/sass/bulma_styles.scss

```diff
@@ -1,2 +1,26 @@
 @charset "utf-8";
+
+/* GLOBAL VARIABLES */
+$hyper: #3A10E5;
+$white: #FFFFFF;
```

Comment:
I _think_ the default `$white` for Bulma is the same as this, so no need to override it.  Please correct me if I'm wrong.